### ISSUE TITLE
Submit password to start up when hitting enter

### ISSUE THREAD

**Issue by cincospenguinos** [cincospenguinos/MinecraftStartup#15]
Right now when I type in my username and password and hit enter nothing happens. A nice quality of life improvement would be submitting my name and password upon hitting enter.

**Comment by Korak-997** [comment]
Hi ,
Can you please provide me some more information ? maybe i can work on it :)

**Comment by cincospenguinos** [comment]
You bet! When the application is running and you reach the landing page, you can type in your username and password to start the minecraft server. As of right now, only the button on the page submits the form. A nice improvement would be allowing the user to hit the enter key to submit the form instead.

The necessary code to change [is in this directory somewhere](https://github.com/cincospenguinos/MinecraftStartup/tree/master/rails_app/app/javascript/components/signup_app) (it's been a while since I've worked on this project.) It should be a relatively straightforward change--add a test using React testing library to assert that hitting enter is a valid way to submit the form, and then add a small listener on the form to submit the information on enter press.

Let me know if you need help getting the project running on your local machine.

**Comment by Korak-997** [comment]
@cincospenguinos  thanks for explanation , i would look forward to see if i can solve it. While i did not work with React yet so it may take me some time to be able to make the changes. but yet i would love to try . So if anyone else suggested changes is better to accept it so you will not wait too long till i will be able to solve the issue.

**Comment by cincospenguinos** [comment]
@Korak-997 yeah man take your time. Go ahead and submit a PR when you have it in place

**Comment by cincospenguinos** [comment]
@Korak-997 did you ever work through this task? Do you need any help with it?
